src/main/java/com/stripe/model/EventDataObjectDeserializer.java

```diff
@@ -75,12 +74,13 @@
   }
 
   /**
-   * Gets data event object, in favor of the deprecated {@link EventData#getObject()}.
-   * When non-, the deserialized {@code StripeObject} preserves high data integrity because of
-   * correspondence between schema of the API response and the model class (the underlying
-   * concrete class for abstract {@code StripeObject}) schema. This is when
-   * {@link Event#getApiVersion()} matches {@link Stripe#API_VERSION}.
-   * @return stripe object that fully represent its original raw JSON response.
+   * Gets optional of data event object, in favor of the deprecated {@link EventData#getObject()}.
```

Comment:
I'd explicitly indicate that "optional" here refers to the type `Optional<T>` because otherwise the the grammar doesn't make sense.

Reply:
==. Maybe `{@link java.util.Optional}` would work? Otherwise `{@code Optional<T>}` is probably explicit enough too.